app/src/playStore/java/org/fossasia/susi/ai/chat/YouTubeActivity.kt

```diff
@@ -27,22 +27,29 @@ class YouTubeActivity : YouTubeBaseActivity(), YouTubePlayer.OnInitializedListen
     lateinit var playbackEventListener: MyPlaybackEventListener
     private var youtubeId: String = ""
     lateinit var playerView: YouTubePlayerView
+    var apikey = ""
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.youtube_player)
 
-        if (BuildConfig.YOUTUBE_API_KEY.equals("YOUR_API_KEY")) {
+
+        val ai = this.packageManager.getApplicationInfo(this.getPackageName(), PackageManager.GET_META_DATA)
+        val bundle = ai.metaData
```

Comment:
Don't create variables which aren't used

Reply:
@iamareebjamal bundle is used , should I eliminate it ?

Reply:
Now I have eliminated it.